testing/run-hunt-results-comparison.ps1

```diff
@@ -0,0 +1,27 @@
+$tests = Import-Csv .\AtomicTestsResults.csv
+[xml]$results = Get-Content .\BLUESPAWNHuntResults.xml
+
+$TotalTests = 0
+$TestsPassed = 0
+
+ForEach($Technique in ($tests | ForEach-Object { $_.Technique } | get-unique)) {
+	$TotalTests += 1
+
+	$TechniqueTests = $tests | Where-Object { $_.Technique -eq $Technique }
+	$TechniqueTestCount = (($TechniqueTests | Measure).count)
+	
+	$TechniqueResults = $results.bluespawn.hunt | Where-Object { $_.name -like "$Technique*" }
+	$TechniqueDetectionCount = (($TechniqueResults.detection | Measure).count)
+	
+	if($TechniqueDetectionCount -ge $TechniqueTestCount) {
+		Write-Host "${TechniqueTestCount}/${TechniqueTestCount} Tests for Technique ${Technique}: PASSED"
```

Comment:
the duplicating of TechniqueTestCount was intentional because sometimes we will generate more detections than there are tests for a specific technique. These tests are only supposed to be good indicator, not a perfect result